### ISSUE TITLE
Naive pluralization

### ISSUE THREAD

**Issue by vbardales** [byteclubfr/js-hal#3]
I know that pluralization can be disabled, but instead of doing naive pluralization, I found an interesting module which can help you to less naively pluralize :
https://github.com/pksunkara/inflect

**Comment by naholyr** [comment]
Yeah, well, it may have been three years :') but thanks for the info, alas it's quite a big dependency for the client. I may extract the pluralization parts from this module, as he does it well, but won't include it as-is.
